src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+// learn how to use array methods like .filter .map .some .every .find .reduce
+// avoid using loop and forEach
+// replace `if ()` statement with &&, || or ?:
+// without nesting
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const diedInCentury = men.filter(man =>
+    Math.ceil(man.died / 100) === century);
```

Comment:
What if century is not passed? You should check it. 
It would be great to call filter method only once and use the ternary operator inside its callback